src/renderer/tvgShape.h

```diff
@@ -209,7 +209,7 @@ struct Shape::Impl : Paint::Impl
         renderFlag |= RenderUpdateFlag::Stroke;
     }
 
-    void strokeTrim(float begin, float end, bool simultaneous)
+    void strokeTrim(float begin, float end, bool simultaneous, bool fillTrim = false)
```

Comment:
Probably, you can add like this:
```
void trimpath(TrimPath& trimpath)
{
}
```

Reply:
I can change the name, but I don't see what is the TrimPath that I should use as an arg. here begin/end and simultaneous are values passed by the api.